lib/sfn-lambda/inject.rb

```diff
@@ -28,11 +28,11 @@ def _lambda(*fn_args)
         new_fn.properties.handler fn_handler
         new_fn.properties.runtime lookup[:runtime]
         new_fn.properties.function_name fn_function_name || fn_name
+        new_fn.properties.role fn_role
```

Comment:
Moving this up so we can set the role even when the lambda function can't be inlined